client/src/components/CreateCard/cardContext.js

```diff
@@ -80,13 +98,10 @@ const CardProvider = props => {
           description,
           tag
         };
-        authFetch(
-          `dashboards/${dashboardId}/columns/${columnId}/tasks/${task}`,
-          {
-            method: "PUT",
-            body: JSON.stringify(updatedTask)
-          }
-        )
+        authFetch("", {
```

Comment:
I'm not sure if its intentional.
this fetch point route ""  to update Task.

Reply:
yes its intentional since url already points to the route